packages/os/os.spec

```
@@ -425,6 +429,11 @@ Summary: XFS progs cli
 %description -n %{_cross_os}xfscli
 %{summary}.
 
+%package -n %{_cross_os}nvidia-migmanager
+Summary: Manages nvidia-migmanager
```

```suggestion
Summary: Manages NVIDIA MIG configuration
```

(otherwise it sounds like it's managing itself)